lib/reverse_sentence.rb

```diff
@@ -1,6 +1,29 @@
 # A method to reverse the words in a sentence, in place.
-# Time complexity: ?
-# Space complexity: ?
+# Time complexity: O(n)
+# Space complexity: O(n) - we're only making a new array called words_arr that's dependent on the size of my_sentence
 def reverse_sentence(my_sentence)
```

Comment:
👍 , could you do it without making the array and just swapping letters in the string directly?